mmdeploy/backend/openvino/onnx2openvino.py

```diff
@@ -55,7 +76,10 @@ def get_output_model_file(onnx_path: str, work_dir: str) -> str:
 
 
 def onnx2openvino(input_info: Dict[str, Union[List[int], torch.Size]],
-                  output_names: List[str], onnx_path: str, work_dir: str):
+                  output_names: List[str],
+                  onnx_path: str,
+                  work_dir: str,
+                  deploy_cfg: Union[None, str, mmcv.Config] = None):
```

Comment:
Hi, @SemyonBevzuk. Could the argument `deploy_cfg` be replaced by the argument of `mo_args`? The latter may require less for users.

Reply:
Hi!
Yes, it's probably better to pass only the necessary information for function `onnx2openvino` to work. I will fix this.